tensorflow_blade/workspace0.bzl

```diff
@@ -50,6 +53,51 @@ def _tf_blade_repositories():
         register_preinstalled_tools = True,  # just use the pre-installed.
     )
 
+    maybe_http_archive(
+        name = "rules_java",
+        sha256 = "f5a3e477e579231fca27bf202bb0e8fbe4fc6339d63b38ccb87c2760b533d1c3",
+        strip_prefix = "rules_java-981f06c3d2bd10225e85209904090eb7b5fb26bd",
+        urls = ["http://pai-blade.oss-cn-zhangjiakou.aliyuncs.com/bazelbuild/rules_java/archive/981f06c3d2bd10225e85209904090eb7b5fb26bd.tar.gz"],
+    )
+
+    # mkldnn cmake external rules
+    maybe_http_archive(
+        name = "mkl_static",
+        build_file = "@org_third_party//bazel/mkldnn:mkl_static.BUILD",
+        sha256 = "b0f4f03c5a2090bc1194f348746396183cfb63a5a379d6e86f7fa89006abe28b",
+        urls = [
+            "https://hlomodule.oss-cn-zhangjiakou.aliyuncs.com/mkl_package/mkl-static-2022.0.1-intel_117.tar.bz2",
+            "https://hlomodule.oss-cn-zhangjiakou.aliyuncs.com/mkl_package/mkl-static-2022.0.1-intel_117.tar.bz2",
+        ],
+    )
+
+    maybe_http_archive(
+        name = "mkl_include",
+        build_file = "@org_third_party//bazel/mkldnn:mkl_include.BUILD",
+        sha256 = "3df729b9fa66f2e1e566c70baa6799b15c9d0e5d3890b9bd084e02299af25002",
+        urls = [
+            "https://hlomodule.oss-cn-zhangjiakou.aliyuncs.com/mkl_package/mkl-include-2022.0.1-h8d4b97c_803.tar.bz2",
+            "https://hlomodule.oss-cn-zhangjiakou.aliyuncs.com/mkl_package/mkl-include-2022.0.1-h8d4b97c_803.tar.bz2",
+        ],
+    )
+
+    # for aarch64 related acl library
+    tf_http_archive(
```

Comment:
No strong opinion but I prefer to depend on tf as less as possible. this a normal `http_archive` be here?

Reply:
I think it may need some extra work here, since the built-in `patch` mechanism in bazel doesn't work for the current patch format.